tests/functional/utils/chart_certification.py

```diff
@@ -289,7 +292,7 @@ def __post_init__(self) -> None:
                     f'HEAD:refs/heads/{current_branch}', '-f')
 
         pretty_test_name = self.test_name.strip().lower().replace(' ', '-')
-        base_branch = f'{pretty_test_name}-{current_branch}' if pretty_test_name else f'test-{current_branch}'
+        base_branch = f'{self.uuid}-{pretty_test_name}-{current_branch}' if pretty_test_name else f'{self.uuid}-test-{current_branch}'
```

Comment:
uuids are very big strings is it necessary for uniqueness?

Reply:
see previous comment :)